org.palladiosimulator.analyzer.slingshot.behavior.systemsimulation.data/src/org/palladiosimulator/analyzer/slingshot/behavior/systemsimulation/entities/seff/SEFFInterpretationContext.java

```diff
@@ -176,6 +188,7 @@ public Builder withParent(final SEFFInterpretationContext parent) {
 		}
 
 		public Builder withCallOverWireRequest(final CallOverWireRequest callOverWireRequest) {
+			assert callOverWireRequest == null || callOverWireRequest.getReplyTo().isEmpty();
```

Comment:
Because a SEFF must not be called by a reply-cowr. I.e. if i check whether replyTo is empty, i ensure, that i only use cowr that are actually calling a SEFF. 

basically, it's just a check to prevent mishaps in case we change code later on.